### PR TITLE
Sprint 12 update documentation

### PR DESCRIPTION
**Pull request by ktp242**
This is the PR to update `RELEASES.md` and version for all the changes we have made for Sprint 12. It should be merged at the last, after all the PRs for Sprint 12 tickets are merged. Please feel free to comment the contents if anything needs to be updated and if any change is missing.

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+## Tag v1.11.0
+* Add CD/CI pipline for deployment.
+* Update styles focus outline for better accessibility performance.
```
Comment (EdwinGuzman) [inline RELEASES.md]: `styles for focus ...`. You don't need the word `performance`.

File: RELEASES.md [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+* Update the structure of location and division pages to remove duplicated links.
+* Update `<main>` tag for better accessibility performance.
+* Update the texts for `See more` buttons on location and division pages.
+* Updated the HTML elements of breadcrumbs as `<nav>`.
```
Comment (EdwinGuzman) [inline RELEASES.md]: `Update the HTML element structure that make up the breadcrumbs component.`

Also, you interchange `Update` and `Updated`. Just one is preferred to keep the tense the same.

File: RELEASES.md [hunk 3]
```diff
@@ -1,3 +1,14 @@
...
+* Updated the contents and add the heading for `Ask NYPL` sections among all the pages.
```
Comment (EdwinGuzman) [inline RELEASES.md]: ~~among~~ ` in all the pages`.

File: RELEASES.md [hunk 4]
```diff
@@ -1,3 +1,14 @@
...
+* Remove the abbreviations for days and months.
+* Update the headings of different sections from all caps to only first character uppercase.
```
Comment (EdwinGuzman) [inline RELEASES.md]: `from all caps to capitalized.`

**Comment by EdwinGuzman** [comment]
Some minor comments.

File: README.md [hunk 5]
```diff
@@ -139,6 +139,6 @@ Run `Locations App` in the correct environment, and the updates should be applie
...
+We can test how the CSS min files of `NYPLBase` work on `Locations App` directly. First, disable the current stylesheet from Cloudfront in `views/index.erb` by removing or commenting out the line, `<link rel="stylesheet" href="<%= settings.env_config['nyplbase'] %>" type="text/css" media="screen" />`. Second, add `<link rel="stylesheet" href="css/nyplbaseTest.min.css" type="text/css" />` in the same file, `view/index.erb`. Last, create a new file named `nyplbaseTest.min.css` in the folder `public/css` and paste the contents from `nyplbaseTest.min.css` to this folder. To compile the correct `nyplbaseTest.min.css` at `NYPLBase`, please check its documentation.
```
Comment (ktp242) [inline README.md]: Not sure why git marked the whole thing lol but I actually only updated from `view/index.erb` to `views/index.erb`

File: RELEASES.md (hunk 2 above)
Comment (holingpoon) [reply RELEASES.md]: I agree with @EdwinGuzman comment on grammar. Think along the lines of either

- This release update**d**/add**ed** X
or
- This release update**s**/add**s** X

Whichever sounds better when you read out loud, just stick with that one convention.

File: RELEASES.md (hunk 4 above)
Comment (holingpoon) [reply RELEASES.md]: Are we talking about capitalizing every first letter of the words in the title? If so, `only first character uppercase` should change to `title case`.

**Pull request merged**